opencensus/proto/stats/metrics/metrics.proto

```diff
@@ -215,12 +215,30 @@ message DistributionValue {
     // The number of values in each bucket of the histogram, as described in
     // bucket_bounds.
     int64 count = 1;
-
-    // TODO: Add support for exemplars.
   }
 
   // If the distribution does not have a histogram, then omit this field.
   // If there is a histogram, then the sum of the values in the Bucket counts
   // must equal the value in the count field of the distribution.
   repeated Bucket buckets = 6;
+
+  // Exemplars are example points that may be used to annotate aggregated
+  // Distribution values. They are metadata that gives information about a
+  // particular value added to a Distribution bucket.
+  message Exemplar {
+    // Value of the exemplar point.
+    int64 value = 1;
```

Comment:
Shouldn't this be a double?

Reply:
I think this `value` is just the bucket count so it should also be int64.

Reply:
Took a second thought, sorry I got this wrong at first. `value` should be the actual value point instead of bucket count, so it should be of type `double`.